app/src/main/java/org/wikipedia/analytics/eventplatform/TimedEvent.kt

```diff
@@ -0,0 +1,36 @@
+package org.wikipedia.analytics.eventplatform
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
+
+@Serializable
+open class TimedEvent(@Transient private val streamName: String = "") : Event(streamName) {
+
+    @Transient
+    private var startTime = 0L
+
+    @Transient
+    private var pauseTime = 0L
+
+    @Transient
+    val duration = System.currentTimeMillis() - startTime
```

Comment:
@sharvaniharan  this field is never updated, and will therefore always be incorrect. You probably intended for this to be a `get()` property?

Reply:
Thank you.. fixed here: #3237